feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -302,6 +302,11 @@ func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string)
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrf).Config(), i)
 			nip := gnmi.OC().NetworkInstance(vrf)
 			fptest.LogQuery(t, "nonDefaultNI", nip.Config(), gnmi.GetConfig(t, dut, nip.Config()))
+		} else if vrf == *deviations.DefaultNetworkInstance {
+			i := d.GetOrCreateNetworkInstance(vrf)
+			i.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE
+			i.Enabled = ygot.Bool(true)
```

Comment:
With the changes in https://github.com/openconfig/featureprofiles/commit/bac3c1844543afcb133401c77d208d3d3f2b90ee, we should remove this line?

Reply:
Also update line#300

Reply:
@sachendras  could you please paste the line#300, as per current code Line#300 is i.Enabled = ygot.Bool(true).